python/paddle/fluid/tests/unittests/test_dist_mnist.py

```diff
@@ -37,10 +38,30 @@ def test_se_resnext(self):
 class TestDistMnistAsync(TestDistBase):
     def _setup_config(self):
         self._sync_mode = False
+        self._use_reduce = False
 
     def test_se_resnext(self):
         self.check_with_place("dist_mnist.py", delta=200)
 
 
+# FIXME(typhoonzero): enable these tests once we have 4
+# 4 GPUs on CI machine, and the base class should be updated.
+#
+# class TestDistMnist2x2ReduceMode(TestDistBase):
```

Comment:
why we need 4 GPU for this test?

Reply:
Reduce mode need at least 2GPUs per trainer.